packages/@aws-cdk/aws-apigatewayv2/lib/common/domain-name.ts

```diff
@@ -1,8 +1,31 @@
 import { ICertificate } from '@aws-cdk/aws-certificatemanager';
+import { IBucket } from '@aws-cdk/aws-s3';
 import { IResource, Resource, Token } from '@aws-cdk/core';
 import { Construct } from 'constructs';
 import { CfnDomainName, CfnDomainNameProps } from '../apigatewayv2.generated';
 
+
+/**
+ * The minimum version of the SSL protocol that you want API Gateway to use for HTTPS connections.
+ */
+export enum SecurityPolicy {
+  /** Cipher suite TLS 1.0 */
+  TLS_1_0 = 'TLS_1_0',
+
+  /** Cipher suite TLS 1.2 */
+  TLS_1_2 = 'TLS_1_2',
+}
+
+/**
+ * Endpoint type for a domain name.
+ */
+export enum EndpointType {
```

Comment:
I understand this is for V2, but the V2 API still supports EDGE here (so a get-domain-name for a V1-created domain name works); we should be including that here

Reply:
+1

Reply:
We don't get v1 domain names when we make a call using the v2 cli. It's the other way around, isn't it? Or am I misunderstanding something?